dashboard/scripts/archive/extra_level_files_for_dsl_defined_levels

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+# Find and remove extra .level files for dsl defined levels
+require_relative '../../config/environment'
+
+DSL_TYPES = %w(TextMatch ContractMatch External Match Multi)
+DSL_TYPES.map(&:constantize).each do |c|
+  c.all.each do |level|
+    # these should be dsl levels and should not have .level files
+    level_file = LevelLoader.level_file_path(level.name)
+    if File.exist? level_file
+      puts `ls -l '#{level_file}'`
```

Comment:
this is not something that we will do a lot (hopefully).
@joshlory made a change to not write these duplicate files: https://github.com/code-dot-org/code-dot-org/commit/f93d54db72579ffaaec46ee76301607b4235159e
so this should be a one time cleanup...